### PR TITLE
Add certs during installation

### PR DESCRIPTION
**Pull request by MrDarthShoe**
Resolves https://github.com/golemfactory/yagna/issues/2605

### CODE REVIEW

File: golem_cli/src/command/provider.rs [hunk 1]
```diff
@@ -302,6 +302,36 @@ impl YaProviderCommand {
...
+        if output.status.success() {
+            Ok(())
+        } else {
+            let output = String::from_utf8_lossy(&output.stderr);
+            Err(anyhow::anyhow!("{}", output))
+        }
+    }
+
+    #[allow(dead_code)]
     pub async fn add_certs<I, S>(self, certs: I) -> anyhow::Result<()>
```
Comment (staszek-krotki) [inline golem_cli/src/command/provider.rs:335]: If this function is not used is it really worth keeping it here?

File: golem_cli/src/manifest.rs [hunk 2]
```diff
@@ -35,40 +35,48 @@ pub async fn manifest_bundle(command: ManifestBundleCommand) -> Result<i32> {
 }
 
 pub async fn add_manifest_bundle(path: String) -> Result<i32> {
-    add_certs(&path).await?;
     add_whitelisted_domains(&path).await?;
+    set_rules(&path).await?;
```
Comment (staszek-krotki) [inline golem_cli/src/manifest.rs:39]: Minor aesthetics: It is probably only a matter of preference but I'd rather get rid of the `set_rules` function and would put its content directly here - this make it 0.1 sec faster to read.  
```suggestion
    set_audited_payload_rules(&path).await?;
    set_partner_rules(&path).await?;
```

File: golem_cli/src/manifest.rs [hunk 3]
```diff
@@ -35,40 +35,48 @@ pub async fn manifest_bundle(command: ManifestBundleCommand) -> Result<i32> {
...
-    let certs = directory
-        // canonicalize first to resolve symlinks
-        .map(|path| path.and_then(|p| p.path().canonicalize()))
-        // this will ignore paths that has errors (for example broken symlinks)
-        .filter(|path| path.as_ref().map(|p| p.is_file()).unwrap_or(false))
-        .map(|path| path.unwrap())
-        .collect::<Vec<_>>();
+async fn set_rules(path: &String) -> Result<()> {
+    set_audited_payload_rules(path).await?;
+    set_partner_rules(path).await?;
```
Comment (staszek-krotki) [inline golem_cli/src/manifest.rs:46]: This could then be removed
```suggestion
```

File: golem_cli/src/command/provider.rs (hunk 1 above)
Comment (MrDarthShoe) [reply golem_cli/src/command/provider.rs]: leftover

**Review by staszek-krotki — APPROVED** [review]
Looks good!

**Pull request merged**